tool/tctl/common/bots_command.go

```diff
@@ -102,6 +110,23 @@ func (c *BotsCommand) Initialize(app *kingpin.Application, config *servicecfg.Co
 	c.botsUpdate.Flag("add-roles", "Adds a comma-separated list of roles to an existing bot.").StringVar(&c.addRoles)
 	c.botsUpdate.Flag("set-logins", "Sets the bot's logins to the given comma-separated list, replacing any existing logins.").StringVar(&c.setLogins)
 	c.botsUpdate.Flag("add-logins", "Adds a comma-separated list of logins to an existing bot.").StringVar(&c.addLogins)
+
+	c.botsInstance = bots.Command("instance", "Manage bot instances.").Alias("instances")
```

Comment:
For `tctl bots` the plural version is the actual command and the singular version is an alias, but for `tctl bots instance` we do the reverse.

Not a big deal but might as well be consistent, no?